### DISCUSSION TITLE
Curious about col order

### DISCUSSION THREAD

**Discussion by fotoflo** [prisma/prisma#17552] [Q&A] [answered]
Hello! Maybe this is a wierd question - how are the cols ordered when I query the database with a regular select?

I have a schema...

```
model reportingBatch {
  id                      BigInt                    @id @unique(map: "id") @default(autoincrement()) @db.UnsignedBigInt
  one                Int?                      @db.Int
  two             Int?                      @db.Int
  three              Int?                      @db.Int
  four            Int?                      @db.Int
```

Often i'll get back the cols in random orders if i query in MySQL workbench (or from the CLI)
![image](https://user-images.githubusercontent.com/1642357/214765190-33670f21-fbe4-4910-a153-f4a327686b01.png)

Just wondering how this order is determined?

**Comment by nurul3101** [comment, marked as answer]
Hello @fotoflo 👋 

I believe the columns are typically returned in the order they are listed in the SELECT statement.
If you do not specify an order for the columns, they will be returned in the order they appear in the table.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
